Jantimizer/ExperimentSuite/UserControls/TestRunner.xaml.cs

```diff
@@ -38,18 +38,20 @@ public partial class TestRunner : UserControl
         public SuiteData RunData { get; }
         public FileInfo SettingsFile { get; private set; }
         public FileInfo SetupFile { get; private set; }
+        public FileInfo? DataInsertsFile { get; private set; }
```

Comment:
Could it be an idea to use this for the old tests as well, they seem to also have inserts in their respective setups